### ISSUE TITLE
Resistences are neither saved nor correctly loaded localy

### ISSUE THREAD

**Issue by joaoruifontes** [codyshepherd/dm-tools#12]

**Comment by codyshepherd** [comment]
Thanks for dropping this issue! Can you describe a series of steps that reliably lead to this behavior, and also identify what you expect to happen vs what is actually happening?

Can you also note whether you're using this with `pip install` or via the Snap, and which version you are using?

**Comment by joaoruifontes** [comment]
TL;DR: The issue I was experiencing is no longer present. So all good and sorry for the trouble 😅.

The title reads what I experienced and when I read the "save locally" function in the code I thought I missed the save resistances part. I was trying to save an encounter with an Invisible Stalker and it's resistances were not saving. There were no "resistances" tags on the file for the rest of the characters being saved either. However, next day, I had a session and when I added a dragon and had to stop early it's resistances where present.

So either I updated something and I forgot and that solved it or... ¯\_(ツ)_/¯

Anyway feel free to close that issue on my end.
